core/embed/rust/src/ui/layout_delizia/flow/set_brightness.rs

```diff
@@ -38,19 +36,17 @@ impl FlowController for SetBrightness {
 
     fn handle_swipe(&'static self, direction: Direction) -> Decision {
         match (self, direction) {
-            (Self::Slider, Direction::Up) => Self::Confirm.swipe(direction),
-            (Self::Confirm, Direction::Down) => Self::Slider.swipe(direction),
-            (Self::Confirmed, Direction::Up) => self.return_msg(FlowMsg::Confirmed),
+            (Self::Slider, Direction::Up) => {
+                let _ = storage::set_brightness(BRIGHTNESS.load(Ordering::Relaxed));
```

Comment:
Maybe use `unwrap!(...)` here as well?
(we probably don't to ignore storage errors)